### PR TITLE
MAT-8163 Remove Feature Flags

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-8163](https://jira.cms.gov/browse/MAT-8163)
(Optional) Related Tickets:
https://github.com/MeasureAuthoringTool/madie-util/pull/119
https://github.com/MeasureAuthoringTool/madie-cql-library/pull/173
https://github.com/MeasureAuthoringTool/madie-layout/pull/389
https://github.com/MeasureAuthoringTool/madie-measure/pull/779
https://github.com/MeasureAuthoringTool/madie-editor/pull/454

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/common/draftMeasureDialog/DraftMeasureDialog.tsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ const DraftMeasureDialog = ({ open, onClose, onSubmit, measure, loading }) => {
           />
         </Box>
         <>
-          {featureFlags?.qiCore6 && !measure?.model.includes("QDM") ? (
```
Comment (adongare) [inline src/components/common/draftMeasureDialog/DraftMeasureDialog.tsx:120]: IDE search gives me 14 matches for this flag. I think there are more files that need to be updated.

Comment (ethankaplan) [reply src/components/common/draftMeasureDialog/DraftMeasureDialog.tsx:120]: I'm getting a check for isQICore6, but none for the flag allowing qiCore to appear

Comment (adongare) [reply src/components/common/draftMeasureDialog/DraftMeasureDialog.tsx:120]: sorry, you are right.

**Review by adongare — APPROVED** [review]
